http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -415,6 +422,51 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift](
       _ <- gceRuntimeMonitor.process(runtime.id).compile.drain
     } yield ()
 
+  private[monitor] def handleCreateDiskMessage(msg: CreateDiskMessage, now: Instant)(
+    implicit traceId: ApplicativeAsk[F, AppContext]
+  ): F[Unit] =
+    for {
+      _ <- googleDiskService.createDisk(
+        msg.googleProject,
+        msg.zone,
+        Disk
+          .newBuilder()
+          .setId(msg.diskId.toString)
```

Comment:
I don't think you need to set an id in the create request. This field is output only, meaning Google will generate its own ID.

Reply:
good to know, will update!